src/components/Main/CommandPalette/SearchResultMessageElement.vue

```diff
@@ -143,9 +158,9 @@ const { toggleSpoilerHandler } = useSpoilerToggler()
   gap: 4px 16px;
   padding: 0.5rem 1rem;
   cursor: pointer;
-  &:hover {
-    @include background-secondary;
-  }
+}
```

Comment:
```suggestion
  &:hover:has(.icon:not(:hover)):has(.header:not(:hover)) {
    @include background-secondary;
  }
}
```

JS書かなくてもこんな感じのCSS書けばよさそうです👀